nilearn/image/resampling.py

```diff
@@ -185,7 +186,7 @@ def get_mask_bounds(img):
 
     """
     img = _utils.check_niimg(img)
-    mask = img.get_data()
+    mask = (img.get_data() != 0.)
```

Comment:
> Memory consumption is the same (we need a boolean copy of
> img.get_data()).

Because it is impossible to have booleans in nifti, right? Elsewhere it
would indeed be different.

That just reminded me what is the "right" way to do it in the nilearn
codebase: _utils.numpy_conversions._asarray(arr, dtype=np.bool). This
code does some tricks to avoid a memory copy when converting from int8 to
bool. Philippe wrote that because he was annoyed that we couldn't store
booleans in nifti files.